game_function.php

```diff
@@ -0,0 +1,24 @@
+
+<?php
+
+function printStats(array $result): string
+{
+
+    $game_data = '';
+
+    foreach ($result as $data) {
+        '<div class="game">';
```

Comment:
This string doesn't actually get output anywhere. If you wanted it to be part of your HTML for each game you would also need to concatenate it into $game_data